src/utils/airseeker-validation.ts

```diff
@@ -33,7 +33,7 @@ export const chainsSchema = z.record(chainSchema);
 export const gatewaySchema = z
   .object({
     apiKey: z.string(),
-    url: z.string().url(),
+    url: z.string(),
```

Comment:
Is this change related to any of the issues in the description? I might be missing something but the changes on the left look more correct that the ones on the right 😅

Reply:
The problem is that the operations validator for Airseeker doesn't handle interpolation and therefore they aren't valid URLs.